src/Peachpie.CodeAnalysis/Utilities/FixpointAnalysis.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Pchp.CodeAnalysis;
+using Pchp.CodeAnalysis.Semantics.Graph;
+using Pchp.CodeAnalysis.Symbols;
+using Pchp.CodeAnalysis.Utilities;
+
+namespace Peachpie.CodeAnalysis.Utilities
+{
+    /// <summary>
+    /// Interface to define custom intraprocedural fixpoint analyses.
+    /// </summary>
+    /// <typeparam name="TState">Flow state to propagate throughout the graph.</typeparam>
+    internal interface IFixPointAnalysisContext<TState>
```

Comment:
why is it in `Utilities` ?

Reply:
I wanted it to be understood as a separate lightweight tool, so I didn't put it to `FlowAnalysis` folder next to the classes used in the main analysis phase. But if it makes more sense (it's true that it's dependent on the CFG), I can move it there.

Reply:
Removed in 2de4d15c2434bddddd8037aa257f199b82a3d7db.